dashboard/app/views/layouts/application.html.haml

```diff
@@ -14,6 +14,10 @@
     = javascript_include_tag "js/#{js_locale}/common_locale"
     %script{src: minifiable_asset_path('js/essential.js')}
     %script{src: minifiable_asset_path('js/code-studio-common.js')}
+
+    //Included early to prevent JS Errors from being sent to New Relic
+    %script{src: minifiable_asset_path('js/layouts/application.js')}
+
```

Comment:
Or alternatively, could we even just make it so that this ends up IN essential I wonder